docs/source/usersguide/geometry.rst

```diff
@@ -0,0 +1,353 @@
+    | :math:`x`-axis       | - R^2(x-x_0)^2 = 0`          |                           |
+    +----------------------+------------------------------+---------------------------+
+    | Cone parallel to the | :math:`(x-x_0)^2 + (z-z_0)^2 | :class:`openmc.YCone`     |
+    | :math:`y`-axis       | - R^2(y-y_0)^2 = 0`          |                           |
+    +----------------------+------------------------------+---------------------------+
+    | Cone parallel to the | :math:`(x-x_0)^2 + (y-y_0)^2 | :class:`openmc.ZCone`     |
+    | :math:`z`-axis       | - R^2(z-z_0)^2 = 0`          |                           |
+    +----------------------+------------------------------+---------------------------+
+    | General quadric      | :math:`Ax^2 + By^2 + Cz^2 +  |  :class:`openmc.Quadric`  |
+    | surface              | Dxy + Eyz + Fxz \\+Gx + Hy + |                           |
+    |                      | Jz + K = 0`                  |                           |
+    +----------------------+------------------------------+---------------------------+
+
+Each surface is characterized by several parameters. As one example, the
+parameters for a sphere are the :math:`x,y,z` coordinates of the center of the
+sphere and the radius of the sphere. All of these parameters can be set either
+as optional keyword arguments to the class constructor or via attributes::
+
+  sphere = openmc.Sphere(R=10.0)
+
+  # ..or..
+  sphere = openmc.Sphere()
+  sphere.r = 10.0
+
+Once a surface has been created, half-spaces can be obtained by applying the
+unary ``-`` or ``+`` operators, corresponding to the negative and positive
+half-spaces, respectively. For example::
+
+   >>> sphere = openmc.Sphere(R=10.0)
+   >>> inside_sphere = -sphere
+   >>> outside_sphere = +sphere
+   >>> type(inside_sphere)
+   <class 'openmc.surface.Halfspace'>
+
+Instances of :class:`openmc.Halfspace` can be combined together using the
+Boolean operators ``&`` (intersection), ``|`` (union), and ``~`` (complement)::
+
+  >>> inside_sphere = -openmc.Sphere()
+  >>> above_plane = +openmc.ZPlane()
+  >>> northern_hemisphere = inside_sphere & above_plane
+  >>> type(northern_hemisphere)
+  <class 'openmc.region.Intersection'>
+
+For many regions, a bounding-box can be determined automatically::
+
+  >>> northern_hemisphere.bounding_box
+  (array([-1., -1., 0.]), array([1., 1., 1.]))
+
+Boundary Conditions
+-------------------
+
+When a surface is created, by default particles that pass through the surface
+will consider it to be transmissive, i.e., they pass through the surface
+freely. If your model does not extend to infinity in all spatial dimensions, you
+may want to specify different behavior for particles passing through a
+surface. To specify a vacuum boundary condition, simply change the
+:attr:`Surface.boundary_type` attribute to 'vacuum'::
+
+   outer_surface = openmc.Sphere(R=100.0, boundary_type='vacuum')
+
+   # ..or..
```

Comment:
I personally like the comment "This is equivalent" that you previously used more than "..or..", though it might be useful (esp. for users who are non-native English speakers) to make these comments even more explicit as "This is equivalent to the code block above".